api/src/API/Controllers/RaceResultsController.cs

```diff
@@ -46,7 +46,24 @@ public async Task<IActionResult> GetAllRaceResults(string orgId)
             IEnumerable<string> memberIds = members.Values.Select(member => member.Id.ToString());
 
             RaceResultContainerClient raceResultContainer = containerProvider.RaceResultContainer;
-            IEnumerable<RaceResult> raceResults = await raceResultContainer.GetRaceResultsForMembersAsync(memberIds);
+
+            string startDateParam = Request.Query["startDate"];
```

Comment:
We should use asp.net model binding for these values

https://docs.microsoft.com/en-us/aspnet/core/mvc/models/model-binding?view=aspnetcore-6.0

Reply:
I think model binding isn't what we want here, since we're not directly querying for a property. We also only have 1 property with type DateTime (`RaceResult.Submitted`), and need 2 properties. You cannot have to binding attributes on the same property.

Instead, I found the `FromQuery` attribute we can add to the controller route. I added that.